docs/self-managed/identity/user-guide/roles/add-assign-permission.md

```diff
@@ -11,7 +11,7 @@ Permissions are assigned to [APIs](/self-managed/identity/user-guide/additional-
 [roles](/self-managed/identity/user-guide/roles/add-assign-role.md).
 
 :::note
-Refer to the [components documentation](../../../../components/components-overview.md) to see what permissions are supported or required.
+You can create permissions for your own APIs for granular access control. Permissions granted to a user or M2M application are added to the `permissions.{audience}` claim of the access token.
```

Comment:
@christinaausley I don't like the double `for ... for` in the first sentence but can not find better wording. May be you can help here?

Reply:
Maybe just, " You can create permissions for granular access control over your APIs."?

Reply:
Sounds good 👍